python/ccxt/base/exchange.py

```diff
@@ -226,6 +226,7 @@ class Exchange(object):
     options = None  # Python does not allow to define properties in run-time with setattr
     accounts = None
     positions = None
+    futuresOldSymbolsMap = None
```

Comment:
this is done to isolate the object for two benefits:
- less memory usage
- clean `markets` structure . no duplicate markets. one symbol = one market
- 
https://discord.com/channels/690203284119617602/921046068555313202/1115727489487085710